src/utils/generic.js

```diff
@@ -240,7 +240,7 @@ bu.hasPerm = async (msg, perm, quiet, override = true) => {
                 role = getId(perm);
                 if (role === null) return false;
             };
-            return Array.isArray(role) ?
+            Array.isArray(role) ?
                 role.indexOf(m.id) > -1 :
                 m.id == role;
```

Comment:
??

Reply:
You mentioned it could create a minor merge conflict, so I removed it?
Should I re-add it then 🤔

Reply:
I mean it's not a problem to have merge conflicts as long as we know about them! I thought I was clear enough when I commented this :-/